tests/countries/test_malta.py

```diff
@@ -9,31 +9,121 @@
 #  Website: https://github.com/dr-prodigy/python-holidays
 #  License: MIT (see LICENSE file)
 
-import unittest
-from datetime import date
+from holidays.countries.malta import Malta, MT, MLT
+from tests.common import TestCase
 
-import holidays
 
+class TestMalta(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Malta, years=range(1980, 2077))
```

Comment:
If there are no holiday checks for the entire period, no need to initialize it.

Reply:
Good to know 👀